Ginger/GingerCore/NoSqlBase/GingerCouchbase.cs

```diff
@@ -136,11 +136,11 @@ private string GetBucketName(string inputSQL)
 
         public override void PerformDBAction()
         {
-            if (!Connect())
-            {
-                Act.Error = "Failed to connect to Couchbase DB";
-                return;
-            }
+            //if (!Connect())
+            //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5368638)